type: choice
is_multiple_choice: false
options:
- text: 'Yes, custom exceptions can be chained by declaring the constructor with the
    Throwable argument. '
- text: 'No, this feature is available for the standard exceptions only. '
message_correct: Congratulations!
message_incorrect: Incorrect solution
files:
- name: src/Task.java
  visible: true
  text: |
    // You can experiment here, it wonâ€™t be checked

    public class Task {
      public static void main(String[] args) {
        // put your code here
      }
    }
  learner_created: false
feedback_link: https://hyperskill.org/learn/step/15646#comment
status: Solved
record: -1
